readme.md

```diff
@@ -2,40 +2,11 @@
-| chromeos&#8209;5.10 | [![chromeos-5.10-clang-18 build status](https://github.com/clangbuiltlinux/continuous-integration2/actions/workflows/chromeos-5.10-clang-18.yml/badge.svg)](https://github.com/clangbuiltlinux/continuous-integration2/actions/workflows/chromeos-5.10-clang-18.yml) | [![chromeos-5.10-clang-17 build status](https://github.com/clangbuiltlinux/continuous-integration2/actions/workflows/chromeos-5.10-clang-17.yml/badge.svg)](https://github.com/clangbuiltlinux/continuous-integration2/actions/workflows/chromeos-5.10-clang-17.yml) | [![chromeos-5.10-clang-16 build status](https://github.com/clangbuiltlinux/continuous-integration2/actions/workflows/chromeos-5.10-clang-16.yml/badge.svg)](https://github.com/clangbuiltlinux/continuous-integration2/actions/workflows/chromeos-5.10-clang-16.yml) | [![chromeos-5.10-clang-15 build status](https://github.com/clangbuiltlinux/continuous-integration2/actions/workflows/chromeos-5.10-clang-15.yml/badge.svg)](https://github.com/clangbuiltlinux/continuous-integration2/actions/workflows/chromeos-5.10-clang-15.yml) | [![chromeos-5.10-clang-14 build status](https://github.com/clangbuiltlinux/continuous-integration2/actions/workflows/chromeos-5.10-clang-14.yml/badge.svg)](https://github.com/clangbuiltlinux/continuous-integration2/actions/workflows/chromeos-5.10-clang-14.yml) | [![chromeos-5.10-clang-13 build status](https://github.com/clangbuiltlinux/continuous-integration2/actions/workflows/chromeos-5.10-clang-13.yml/badge.svg)](https://github.com/clangbuiltlinux/continuous-integration2/actions/workflows/chromeos-5.10-clang-13.yml) | [![chromeos-5.10-clang-12 build status](https://github.com/clangbuiltlinux/continuous-integration2/actions/workflows/chromeos-5.10-clang-12.yml/badge.svg)](https://github.com/clangbuiltlinux/continuous-integration2/actions/workflows/chromeos-5.10-clang-12.yml) |     |     |
-
-[![Check clang version](https://github.com/clangbuiltlinux/continuous-integration2/actions/workflows/clang-version.yml/badge.svg)](https://github.com/clangbuiltlinux/continuous-integration2/actions/workflows/clang-version.yml)
-
 ## Useful links
 
 - [Official Kernel Docs](https://www.kernel.org/doc/html/latest/kbuild/llvm.html)
 - [Issue tracker](https://github.com/ClangBuiltLinux/linux/issues)
+- [Build Status](https://github.com/ClangBuiltLinux/continuous-integration2/#readme)
```

Comment:
note: we still retain a link, so "remove build status" is not accurate. More like "replace build status image grid in favor of link"